site/docs/design_tokens/colour.mdx

```diff
@@ -138,6 +138,20 @@ Status colours are reserved exclusively for indicating valid, invalid or informa
 | **Black 20**  | black 90      | **AAA** `13,8 : 1`    | <ColorContrast color="var(--color-black-90)" background="var(--color-black-20)">AAA</ColorContrast> | |
 | **Black 10**  | black 90      | **AAA** `15,6 : 1`    | <ColorContrast color="var(--color-black-90)" background="var(--color-black-10)">AAA</ColorContrast> | |
 
+### Focus colour
+HDS components use a custom focus colour styling instead of the default browser provided style. The default HDS focus style is accessible on light backgrounds. Since HDS does not offer dark mode, when using HDS components on darker backgrounds, it may be required to customize the focus style colour. The WCAG requirement for user interface element contrast is `3 : 1`.
```

Comment:
Lauserakenne vähän vaikea seurata. Vaihtaisin vähän järjestystä:
"Since HDS does not offer dark mode, the focus style colour may require customising when using HDS components on darker backgrounds."